compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -226,6 +225,49 @@ pub struct InliningMap<'tcx> {
     inlines: GrowableBitSet<usize>,
 }
 
+/// Struct to store mono items in each collecting and if they should
+/// be inlined. We call `instantiation_mode` to get their inlining
+/// status when inserting new elements, which avoids calling it in
+/// `inlining_map.lock_mut()`. See the `collect_items_rec` implementation
+/// below.
+struct MonoItems<'tcx> {
+    // Whether the collecting start from the root. The mono
+    // items will not be inlined if so.
+    is_root: bool,
```

Comment:
This comment is not accurate.  The questions is whether we need to compute `instantiation_mode`, not about inlining.
Proposed redaction: "When collecting from the root, we do not need to compute whether items will need to be inlined."
Should this be named `compute_inlining`?

Reply:
Thanks! Such a comment is indeed more accurate